### DISCUSSION TITLE
ФРМО. Есть ли обмен данными с внешними системами?

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#70] [Административная РГ] [answered]
В спецификации ФРМО более 500 полей. Возможно, существует интерфейс взаимодействия, позволяющий загрузить в ФРМО часть этих данных. Если это верно, то следует в первую очередь смоделировать на FHIR данные, участвующие в информационном обмене.

**Comment by iradio** [comment, marked as answer]
Интеграционные профили ФРМО практически полностью покроют атрибутный состав, т.к. решают задачи автоматизации наполнения ФРМО и наполнения смежных системы данными ФРМО.  Таким образом вхождение атрибута в интеграционное взаимодействие не делает его приоритетным с точки зрения работы над профилем. 
Сами профили будут опубликованы по готовности новой версии ФРМО (пока в работе).

**Reply by eugenekogan-spb** [reply]
Ок, спасибо. Будем считать что вопрос решен, и интеграционные профили - не критерий.
Продолжим поиск критериев исходя, видимо, просто из здравого смысла.
